@here/harp-geoutils/lib/projection/MercatorProjection.ts

```diff
@@ -328,6 +328,15 @@ class WebMercatorProjection extends MercatorProjection {
         MathUtils.newVector3(0, 0, -1, result.zAxis);
         return result;
     }
+
+    /** @override */
+    localTangentSpaceFromWorld(worldPos: Vector3Like, result: TransformLike): TransformLike {
```

Comment:
`localTangentSpaceFromWorld` sounds strange to me, how about `localTangetSpaceAtPosition` ?
Alternatively you could overload the signature of localTangentSpace, e.g.
```
localTangentSpace(position: GeoCoordinatesLike | Vector3Like, result: TransformLike) {
    if (!isVector3Like(position)) {
        position = this.projectPoint(position);
   }
   // ...
```

Reply:
I would also prefer to just add an overload for the existing method

Reply:
just saw that you did this already. Should look at the code before commenting next time ;-)